lib/Conversion/ExportVerilog/ExportVerilog.cpp

```diff
@@ -290,7 +300,14 @@ static StringRef getVerilogDeclWord(Operation *op,
   // "automatic logic" values aren't allowed in disallowLocalVariables mode.
   assert((!isProcedural || !options.disallowLocalVariables) &&
          "automatic variables not allowed");
```

Comment:
The patch looks good, but since you're making this an "if" please sink the assert after the if.  This will simplify it to just `assert(!options.disallowLocalVariables && ...`

Reply:
Thanks!